src/ports/analyticsDayData/utils.ts

```diff
@@ -11,6 +12,16 @@ export const getAnalyticsDayDataFragment = () => `
   }
 `
 
+export const getAnalyticsTotalDataFragment = () => `
```

Comment:
This is called _getAnalytics_**Total**_DataFragment_, but it's in the _src/ports/analytics_**Day**_Data/utils.ts_ file, is that correct?

Reply:
updated!